src/data.js

```diff
@@ -1,7 +1,6 @@
 import React from "react"
```

Comment:
Better to remove this Import from react, you don't need it in the new version of react.